app/src/main/java/ch/logixisland/anuto/view/faq/EnemiesAdapter.java

```diff
@@ -0,0 +1,212 @@
+            txt_reward = view.findViewById(R.id.txt_reward);
+            txt_weak_against = view.findViewById(R.id.txt_weak_against);
+            txt_strong_against = view.findViewById(R.id.txt_strong_against);
+        }
+    }
+
+    @Override
+    public int getCount() {
+        return mEnemyProperties.size();
+    }
+
+    @Override
+    public Object getItem(int position) {
+        return mEnemyProperties.get(position);
+    }
+
+    @Override
+    public long getItemId(int position) {
+        return 0;
+    }
+
+    private Bitmap extractSingleBmp(EnemyType enemyType) {
+        int attrId, spriteCount, spriteId;
+
+        switch (enemyType) {
+            case soldier:
+                attrId = R.attr.soldier;
+                spriteCount = 12;
+                spriteId = 0;
+                break;
+            case blob:
+                attrId = R.attr.blob;
+                spriteCount = 9;
+                spriteId = 0;
+                break;
+            case sprinter:
+                attrId = R.attr.sprinter;
+                spriteCount = 6;
+                spriteId = 3;
+                break;
+            case flyer:
+                attrId = R.attr.flyer;
+                spriteCount = 6;
+                spriteId = 4;
+                break;
+            case healer:
+                attrId = R.attr.healer;
+                spriteCount = 4;
+                spriteId = 0;
+                break;
+            default:
+                throw new RuntimeException("Unknown enemy!");
+        }
+
+        Bitmap sheet = BitmapFactory.decodeResource(appContext.getResources(), mTheme.getResourceId(attrId));
+        int spriteWidth = sheet.getWidth() / spriteCount;
+        int spriteHeight = sheet.getHeight();
+
+
+        float aspect = (float) spriteWidth / spriteHeight;
+
+        float newHeight = 120;
+        float newWidth = newHeight * aspect;
+        float scaleHeight = (newHeight) / spriteHeight;
+        float scaleWidth = (newWidth) / spriteWidth;
+        Matrix matrix = new Matrix();
+        matrix.postScale(scaleWidth, scaleHeight);
+
+        Bitmap resizedBitmap = Bitmap.createBitmap(sheet, spriteWidth * spriteId, 0, spriteWidth, spriteHeight, matrix, false);
+        sheet.recycle();
+        return resizedBitmap;
+    }
+
+
+    public View getView(int position, View convertView, ViewGroup parent) {
+        Activity activity = mActivityRef.get();
+
+        if (activity == null) {
+            return convertView;
+        }
+
+        View enemyItemView;
+
+        if (convertView == null) {
+            enemyItemView = LayoutInflater.from(activity).inflate(R.layout.item_enemy, parent, false);
+        } else {
+            enemyItemView = convertView;
+        }
+        EnemyProperties enemyProperties = mEnemyProperties.get(position);
+        ViewHolder viewHolder = new ViewHolder(enemyItemView);
+        String dmp;
+
+        switch (enemyProperties.getEnemyType()) {
```

Comment:
Same argumentation as for the EnemyProperties. Would be better to get the bitmaps directly from the Enemy classes using the EntityRegistry.